app/controllers/api/v1/activities_controller.rb

```diff
@@ -6,6 +6,9 @@ def latest
 
   def just_updated
     versions = Version.just_updated(50)
+    if params[:since].present?
```

Comment:
@dwradcliffe I believe it is already limited by the original call, which limits to 50 records. Or am I overlooking something?